docs/content/Popover.md

```diff
@@ -107,25 +111,75 @@ function CaretSelector(props) {
 render(<PopoverDemo />)
 ```
 
-## Component props
+## Props
 
 ### Popover
 
-| Name     | Type              | Default | Description                                                                    |
-| :------- | :---------------- | :-----: | :----------------------------------------------------------------------------- |
-| as       | String            |  'div'  | Sets the HTML tag for the component.                                           |
-| caret    | String            |  'top'  | Controls the position of the caret. See below for the list of caret positions. |
-| open     | Boolean           |  false  | Controls the visibility of the popover.                                        |
-| relative | Boolean           |  false  | Set to true to render the popover using relative positioning.                  |
-| sx       | SystemStyleObject |   {}    | Style to be applied to the component                                           |
-
-#### Caret Positions
-
-The `caret` prop can be one of the following values: `top`, `bottom`, `left`, `right`, `bottom-left`, `bottom-right`, `top-left`, `top-right`, `left-bottom`, `left-top`, `right-bottom`, or `right-top`.
+<PropsTable>
+  <PropsTableRow
+    name="as"
+    defaultValue="div"
+    type="string"
+    description="Sets the underlying HTML tag for the component"
+  />
```

Comment:
```suggestion
  <PropsTableAsRow
    defaultElementType="div"
  />
```

Reply:
Same comment as before, this suggestion results in an empty row? @colebemis